engine-cockpit/webContent/view/businesscalendar-detail.xhtml

```diff
@@ -20,7 +20,7 @@
     </ui:define>
 
     <ui:define name="topbar-items">
-      <cc:TopBarItem styleClass="editor-link" href="editor.xhtml?file=app-#{managerBean.selectedApplicationName}.yaml" icon="si-pencil" title="#{ivy.cm.co('/businessCalendarDetail/EditAppYamlFile')}" external="false" />
+      <cc:TopBarItem styleClass="editor-link" href="editor.xhtml?file=#{managerBean.selectedApplicationName}/app.yaml" icon="si-pencil" title="#{ivy.cm.co('/businessCalendarDetail/EditAppYamlFile')}" external="false" />
```

Comment:
I recommend to build this link in a bean so that proper url escaping is used, e.g. see:

```
>> User.java
private static String getViewUrl(String securityContext, String name) {
    return UriBuilder.fromPath("userdetail.xhtml")
        .queryParam("system", securityContext)
        .queryParam("name", name)
        .build()
        .toString();
  }
```